config/m4/compiler.m4

```diff
@@ -211,6 +211,18 @@ AC_DEFUN([ADD_COMPILER_FLAG_IF_SUPPORTED],
                              [$5])
 ])
 
+#
+# ADD_COMPILER_FLAGS_IF_SUPPORTED
+# Usage: ADD_COMPILER_FLAGS_IF_SUPPORTED([[flag1], [flag2], [flag3]], [program])
+#
+# The macro checks multiple flags supported by compiler
+#
+AC_DEFUN([ADD_COMPILER_FLAGS_IF_SUPPORTED],
+[
+         m4_foreach([_flag], [$1],
```

Comment:
indentation is weird

Reply:
hmmm, indentation is same as for other function declarations in file, for example:
```
AC_DEFUN([CHECK_DEPRECATED_DECL_FLAG],
[
         AC_MSG_CHECKING([whether $1 overrides deprecated declarations])
         SAVE_CFLAGS="$CFLAGS"
         CFLAGS="$BASE_CFLAGS $CFLAGS $1"
```